### PR TITLE
[13.0][MIG] base_repair_config: Migration to 13.0

### PR DESCRIPTION
**Pull request by marcelsavegnago**

### CODE REVIEW

**Comment by marcelsavegnago** [comment]
cc @rvalyi @eduaparicio

**Comment by rvalyi** [comment]
just a doubt: why this as a separate module and not having this in base_repair?
(I'm not saying it should. I'm just curious about the design).

**Comment by marcelsavegnago** [comment]
> just a doubt: why this as a separate module and not having this in base_repair?
> (I'm not saying it should. I'm just curious about the design).

In fact, I followed the logic applied in base_maintenance and base_maintenance_config. But if you agree, for version 13 I can include this feature in base_repair module without any problem.

https://github.com/oca/maintenance/tree/11.0

**Comment by rvalyi** [comment]
all right no big deal I see no problem this way either.

**Review by rvalyi — APPROVED** [review]

**Comment by marcelsavegnago** [comment]
> all right no big deal I see no problem this way either.

When I migrate to version 14 I will put everything together in base_repair.

**Comment by marcelsavegnago** [comment]
ping @OCA/manufacturing-maintainers

**Comment by marcelsavegnago** [comment]
ping @OCA/manufacturing-maintainers

**Comment by marcelsavegnago** [comment]
ping @OCA/manufacturing-maintainers

**Comment by marcelsavegnago** [comment]
ping @dreispt

**Comment by marcelsavegnago** [comment]
ping @OCA/manufacturing-maintainers

**Comment by marcelsavegnago** [comment]
ping @mileo

**Comment by marcelsavegnago** [comment]
ping @mileo @renatonlima

**Comment by marcelsavegnago** [comment]
ping @OCA/manufacturing-maintainers

**Comment by marcelsavegnago** [comment]
ping @bodedra

**Comment by marcelsavegnago** [comment]
ping @OCA/manufacturing-maintainers

**Comment by marcelsavegnago** [comment]
ping @OCA/manufacturing-maintainers

**Comment by marcelsavegnago** [comment]
ping @OCA/manufacturing-maintainers

**Comment by marcelsavegnago** [comment]
@dreispt You have informed that this PR needs fixing. Do you need any corrections to be made in the migration?

**Comment by dreispt** [comment]
Merge was aborted because of a build error. Please check the build log.

File: base_repair_config/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+{
+    "name": "Base Repair Config",
+    "summary": """
+        Provides general settings for the Maintenance App""",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Escodoo,Odoo Community Association (OCA)",
+    "maintainers": ["marcelsavegnago"],
+    "images": ["static/description/banner.png"],
+    "website": "https://github.com/oca/manufacture",
```
Comment (bodedra) [inline base_repair_config/__manifest__.py]:
```suggestion
    "website": "https://github.com/OCA/manufacture",
```

**Review by bodedra — COMMENTED** [review]
Suggestion. Rebase your PR with latest OCA/manufacture branch and re-run pre-commit one more time.

File: base_repair_config/__manifest__.py (hunk 1 above)
Comment (marcelsavegnago) [reply base_repair_config/__manifest__.py]: @bodera Done.

Comment (bodedra) [reply base_repair_config/__manifest__.py]: You pinged someone else.

**Review by bodedra — APPROVED** [review]
Code review. :+1:

**Pull request merged**